content/en/user-guide/web-application/teams/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Teams"
+tags: [] 
+weight: 2
+description: >
+  LocalStack Organizations/Teams offers a flexible way of managing access roles and permissions for your team members.
+---
+
+Teams on LocalStack allow you to collaborate with members of your organization. You can invite other users to your team and assign them different roles and permissions. It includes advanced role settings such as purchasing subscriptions, managing payment methods, and more by the team administrator. You can purchase a plan with multiple seats for your team, which allows you to distribute the API keys among your team members.
```

Comment:
`It includes advanced role settings such as purchasing subscriptions, managing payment methods, and more, which can be adjusted by the team administrator.`